<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuoteDlg</class>
 <widget class="QDialog" name="QuoteDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddQuoteDlg</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="leTitle"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QPlainTextEdit" name="teContent">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>2</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>1</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>References</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <property name="spacing">
        <number>5</number>
       </property>
       <item row="0" column="0">
        <widget class="QPushButton" name="btAdd">
         <property name="toolTip">
          <string>Add new reference</string>
         </property>
         <property name="icon">
          <iconset resource="MainWindow.qrc">
           <normaloff>:/Images/Add.png</normaloff>:/Images/Add.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+=</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btDel">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>Remove reference</string>
         </property>
         <property name="icon">
          <iconset resource="MainWindow.qrc">
           <normaloff>:/Images/Minus.png</normaloff>:/Images/Minus.png</iconset>
         </property>
         <property name="shortcut">
          <string>Del</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="6">
        <widget class="QListView" name="listView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btSelect">
         <property name="toolTip">
          <string>Select existing reference</string>
         </property>
         <property name="icon">
          <iconset resource="MainWindow.qrc">
           <normaloff>:/Images/FullText.png</normaloff>:/Images/FullText.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+F</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="btViewPDF">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>View PDF</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="MainWindow.qrc">
           <normaloff>:/Images/PDF.png</normaloff>:/Images/PDF.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+R</string>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QPushButton" name="btGotoQuotePage">
         <property name="toolTip">
          <string>Go to Quote page</string>
         </property>
         <property name="icon">
          <iconset resource="MainWindow.qrc">
           <normaloff>:/Images/Quote.png</normaloff>:/Images/Quote.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MainWindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QuoteDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>-9</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QuoteDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>381</x>
     <y>464</y>
    </hint>
    <hint type="destinationlabel">
     <x>360</x>
     <y>-17</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
